.gitlab/ci/.gitlab-ci.on-push.yml

```diff
@@ -236,6 +236,15 @@ xpanse-prepare-testing-bucket:
       optional: true
   script:
     - EXIT_CODE=0
+    - section_start "Check if nightly"
```

Comment:
Maybe something like `section_start "Check if should run servers GA...."` 
I think `check if nightly` is a bit confusing